README.md

```diff
@@ -10,22 +10,106 @@ Digital service mock to claim public money in the event property subsides into m
 |PORT|Port number|no|3002|||
 |POSTGRES_USERNAME|Postgres username|yes||||
 |POSTGRES_PASSWORD|Postgres password|yes||||
+|POSTGRES_DATABASE|Postgres database|no|mine_users|||
+|POSTGRES_HOST|Postgres host|no|mine-support-postgres-users|||

```

Comment:
I've added env vars so the database name and host can be set.
The database and hostname were already used in the Helm chart for setting up the external service for the database connection. 
Moving them to env vars and passing them down from the charts keeps them in once place so they can't get out of sync.
This also helps if you want to test locally, and allows us to set a different database name for PRs in future.